Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM python:alpine3.7
+
+# #Installing requirements for Pillow
+RUN apk --update add libxml2-dev libxslt-dev libffi-dev gcc musl-dev libgcc openssl-dev curl
+RUN apk add jpeg-dev zlib-dev freetype-dev lcms2-dev openjpeg-dev tiff-dev tk-dev tcl-dev
+
+WORKDIR /app
+
+COPY requirements.txt /app
+RUN pip install -r requirements.txt
+
+COPY dev_requirements.txt /app
+RUN pip install -r dev_requirements.txt
```

Comment:
Do we need to add dev requirements if the docker image will only be used for deployments?

Reply:
Thanks. I think I don't need to add `maildev` to the services too. Since it is only for deployment. Is that okay?

Reply:
But it leads to a series of problems. First one is that the application can't find Faker when it starts. 
I removed the initialise command and the seeding command because they were raising errors about things that are Dev dependencies not found. What do you say about this?